ioncore/event.c

```diff
@@ -66,7 +66,7 @@ static void check_signals()
 
 /*{{{ Timestamp stuff */
 
-#define CHKEV(E, T) case E: tm=((T*)ev)->time; break;
+#define CHKEV(E, T) case E: tm=((T*)ev)->time; break; ((void)0)
```

Comment:
In cases where the macro is a statement rather than a block, wouldn't it be neater to just drop the final `;` rather than appending a `((void)0)`?
```suggestion
#define CHKEV(E, T) case E: tm=((T*)ev)->time; break
```

Reply:
Done.